uri/exid-types/RIN.yaml

```diff
@@ -1,13 +1,15 @@
 %YAML 1.2
 ---
+label: RIN
+
 lang: en
 
 type: uri
 
 uri: https://gedcom.io/terms/v7/RIN
 
 specification:
-  - Record ID Number
+  - Automated Record ID
```

Comment:
Here the two files had different values.  Perhaps "Record ID Number" should be moved to the label?